### PR TITLE
Tweaks for session 4 content

### PR DESCRIPTION
**Pull request by thibaudcolas**
Main changes are adding the missing region flag for buckup, and policy to create IAM users.

### CODE REVIEW

File: src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx [hunk 1]
```diff
@@ -23,31 +23,25 @@ We will also be using the AWS CLI. So install that if you haven't already.
...
 
-I'm going to call my user `cocdersofcolouruser`
+I'm going to call my user `codersofcolouruser`
 
 ![](./aws_3.png)
 
-Next you will be asked to add the user to a group. For now, we'll set up a full S3 user.
-
-In a new tab, Locate the groups link in the left menu and click Create new group
+Next you will be asked to add the user to a group. For now, we'll set up a full S3 user. Click "Create group"
```
Comment (thibaudcolas) [inline src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx:36]: It’s possible to do this right from the same screen in a modal, so I’ve reworded the content for that option.

File: src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx [hunk 2]
```diff
@@ -23,31 +23,25 @@ We will also be using the AWS CLI. So install that if you haven't already.
...
 
-Group Name: S3 (you can see I already have s3full)
-
-The next screen will ask you to attach a policy. Search for s3 and find AmazonS3FullAccess, tick it and click next step.
-
-When your group is created, switch back to your IAM user tab
+Group Name: S3
 
-Click the refresh button next to create group.
+In the policies, search for s3 and find "AmazonS3FullAccess", tick it, then search for iam, and tick "IAMFullAccess", and click "Create group".
```
Comment (thibaudcolas) [inline src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx:42]: Was missing "IAMFullAccess", which causes an error when running buckup (https://github.com/torchbox/buckup#usage).

File: src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx [hunk 3]
```diff
@@ -68,10 +62,10 @@ aws configure
 
 You will be asked for your credentials we just viewed. Add those in and just accept defaults for the next items by pressing return.
 
-Now we can run buckup. So in our terminal, run `buckup`
+Now we can run buckup. So in our terminal, run `buckup --region eu-west-2`
```
Comment (thibaudcolas) [inline src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx:65]: `buckup` doesn’t proceed without a region.

File: src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx (hunk 2 above)
Comment (kevinhowbrook) [reply src/docs/wagtail-intro/Session-4-Setting-up-AWS-for-media-files.mdx:42]: v nice catch

**Pull request merged**